tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>Add a sentence above again on what classification task is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>